services/basic_types.proto

```diff
@@ -1415,3 +1415,28 @@ message TokenAssociation {
     TokenID token_id = 1;  // The token involved in the association
     AccountID account_id = 2; // The account involved in the association
 }
+
+/**
+ * Related to EIP-2929 and EIP-2930 addresses can be provided for "warm up" in EVM before the execution of the
+ * transaction in order to save gas.
+ */
+message AccessListEntry {
+
+    oneof entity {
+
+        /**
+         * The address of the account that should be "warmed up"/loaded in the EVM
+         */
+        AccountID accountID = 1;
+
+        /**
+         * The address of the smart contract that should be "warmed up"/loaded in the EVM
+         */
+        ContractID contractID = 2;
+    }
+
+    /**
+     * The storage keys of the smart contract that should be "warmed up"/loaded in the EVM
+     */
+    repeated bytes storageKeys = 3;
```

Comment:
Only contracts have storage.  If we need to separate account and storage IDs I think we should explore only having storage keys when it's a contract, such as creating a separate message and having he oneof being an AccountId or the new combined message.